Diffusion/Common/State.swift

```diff
@@ -126,4 +128,25 @@ class Settings {
             return ComputeUnits(rawValue: current)
         }
     }
+    
+    public func applicationSupportURL() -> URL {
```

Comment:
I think this is fine. Maybe just create another helper for the model location and use it from both `PipelineLoader` and `ImportModelBehavior` for consistency.